### ISSUE TITLE
SQL error: 'Unknown column 'tstamp' in 'field list''

### ISSUE THREAD

**Issue by nicodh** [fsuter/devlog#8]
I see these errors in the sys_log:

SQL error: 'Unknown column 'tstamp' in 'field list'' (tx_devlog_domain_model_entry:NEW5d7b324c3dc35547020303) (msg#1.1.12)

and in fact this column is missing but referenced in TCA:
$TCA['tx_devlog_domain_model_entry']['ctrl']['tstamp'] => 'tstamp'

on the other hand the error does not show up always...??

Happens in release 3.0.4

**Comment by fsuter** [comment]
You're right, that TCA property should not be defined as it does not make any sense.
